lego/apps/events/tests/test_events_api.py

```diff
@@ -419,6 +481,76 @@ def test_event_creation(self):
             for key in ["name", "capacity", "activationDate", "permissionGroups"]:
                 self.assertEqual(res_pools[i][key], expect_pools[i][key])
 
+    def test_event_creation_tba(self):
+        """Test event creation for TBA status type"""
+        self.event_response = self.client.post(_get_list_url(), _test_event_data[2])
+        self.assertEqual(self.event_response.status_code, 201)
```

Comment:
Shouldn't we return 400 if the user tries to set location and pools instead of "failing" silently?

Reply:
I think the backend should just overwrite the input and set the event to be TBA with `pools = []` and return `201`. That way frontend bugs where location and events suddenly retain their value or something won't crash the application.